static/css/daily_location.css

```diff
@@ -0,0 +1,78 @@
+html, body { height: 100%; }
```

Comment:
Most of this file's CSS is identical to daily.css. I think it would make more sense to have a common css file they both share as the added style for the map wouldn't cause any harm to the other tabs.